src/items/predefined.rs

```diff
@@ -241,7 +246,7 @@ fn test_about_metadata() {
 #[derive(Debug, Clone)]
 #[non_exhaustive]
 #[allow(clippy::large_enum_variant)]
-pub(crate) enum PredefinedMenuItemType {
+pub enum PredefinedMenuItemType {
```

Comment:
if we are to expose this type, I think it should be called `PredefinedMenuItemKind`